lib/index.js

```diff
@@ -0,0 +1,6 @@
+let { Given, When, Then } = require('@cucumber/cucumber');
+let lib = require('wdi5-fe-selectors');
+
+Given('we press tile {string}', async function (name) {
```

Comment:
any need for the `this`-scoped `function` as opposed to `async (name) => {`?

Reply:
done - no reason currently